website/blog/2023-05-07-1M-milestone/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Surpassing 1 Million Downloads - A Retrospective and a Look into the Future
+authors: qingyunwu
+tags: [LLM, LLMOps, FLAMLv2]
+---
+
+**TL;DR:**
+* **Celebrating FLAML's milestone: 1 million downloads**
+* **Introducing Large Language Model (LLM) support in the upcoming FLAML v2**
+
+
+This week, FLAML has reached a significant milestone: 1 million downloads. Originating as an intern research project within Microsoft Research, FLAML has grown into an open-source library used widely across the industry and supported by an active community.
+As we celebrate this milestone, we want to recognize the passionate contributors and users who have played an essential role in molding FLAML into the flourishing project it is today. Our heartfelt gratitude goes out to each of you for your unwavering support, constructive feedback, and innovative contributions that have driven FLAML to new heights.
```

Comment:
A big shoutout to our industrial collaborators from Azure Compute, Azure Data (@thinkall), Azure Machine Learning, Microsoft 365, ML.NET, Vowpal Wabbit, Anyscale, Databricks, and Wise; and academic collaborators from MIT, Penn State University, Stevens Institute of Technology, Tel Aviv University, Texas A & M University, University of Manchester, University of Washington, and The Chinese University of Hong Kong etc.

Reply:
Added. thanks!

Reply:
okay, @thinkall, could you double check?

Reply:
> okay, @thinkall, could you double check?

Oh, yes, Azure Data is correct. Moreover, Azure Synapse is even more precise and is at the same organizational level as Azure Machine Learning.

Reply:
Thanks! Updated!

Reply:
As our PM commented today, "Azure Synapse Analytics" is better.

Reply:
Thanks @thinkall ! Updated.